mappings/net/minecraft/server/world/ChunkStorage.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5318 net/minecraft/server/world/ChunkStorage
+	METHOD method_27476 loadChunk (Lnet/minecraft/class_5086;II)Lnet/minecraft/class_3607;
```

Comment:
Maybe `read` ? or is that misleading?

Reply:
I think load fits better since it doesn't always read from disk and if it reads it also applies data fixes